win32/src/interlocked_hl_win32.c

```diff
@@ -268,8 +222,8 @@ IMPLEMENT_MOCKABLE_FUNCTION(, INTERLOCKED_HL_RESULT, InterlockedHL_SetAndWake, i
         /*Codes_SRS_INTERLOCKED_HL_02_017: [ InterlockedHL_SetAndWake shall set address to value. ]*/
         (void)interlocked_exchange(address, value);
 
-        /*Codes_SRS_INTERLOCKED_HL_02_018: [ InterlockedHL_SetAndWake shall call WakeByAddressSingle. ]*/
-        WakeByAddressSingle((void*)address);
+        /*Codes_SRS_INTERLOCKED_HL_02_018: [ InterlockedHL_SetAndWake shall call wake_by_address_single. ]*/
+        wake_by_address_single((void*)address);
```

Comment:
>(void*) [](start = 31, length = 7)

cast shouldn't be needed (I thought we setup the PAL API to avoid casting). Here and below

Reply:
Removed cast. It remained, because I did a replace :-).

---
In reply to: [500455431](https://github.com/Azure/azure-c-pal/pull/42#discussion_r500455431) [](ancestors = 500455431)